indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskQueue.java

```diff
@@ -443,8 +447,13 @@ private void startPendingTaskOnRunner(TaskEntry entry, ListenableFuture<TaskStat
                   e.getMessage()
               );
             }
+            statusUpdatesInQueue.incrementAndGet();
```

Comment:
This should be incremented just before adding items to the `taskCompleteCallbackExecutor`, not here.